### PR TITLE
Issue #269 update actionkit connector docs

### PR DESCRIPTION
**Pull request by rgriff23**
This PR updates connector documentation for the ActionKit class, per issue #269.

Change summary:

- Updated the overview and quickstart in the `*.rst`, aiming to following the examples provided in the issue description as closely as possible
- Fixed a couple typos in the main docstring for the class
- Cleaned up `**kwargs` documentation, making it consistent and correct across functions

All tests passed

### CODE REVIEW

**Comment by rgriff23** [comment]
I based my changes off existing examples, but I'm not sure if my `*.rst` formatting is correct. Is there a good way to check the impact of the changes?

**Comment by shaunagm** [comment]
@rgriff23 there should be a way to build the docs locally and inspect your changes there, but it's not been written down yet (and I personally don't know it). For now the docs just link to [the sphinx guide for rebuilding](https://www.sphinx-doc.org/en/master/usage/quickstart.html) but I wasn't able to get that working myself so I'm not sure how helpful it will be.  Hopefully @eliotst or @SorenSpicknall can get us the right process.

At least on a content level, your updates look _fantastic_.  We finally have our "model connector docs" we can point to and say "make your documentation look like this!"

**Comment by jburchard** [comment]
For creating docs internally you need two packages:

```
Sphinx==1.8.3
sphinx-rtd-theme==0.4.2
```

Then to build do the following:

```
> cd docs
> make html
```

It will generate all of the html locally that you can then inspect. If there are recommendations as to how to better systematize this, I'm open to it. The html files shouldn't be included in the commit though, as they are built on deploy.

File: docs/action_kit.rst [hunk 1]
```diff
@@ -5,19 +5,53 @@ ActionKit
 Overview
 ********
 
-The ActionKit class allows you to interact with an `ActionKit <https://actionkit.com/>`_.
+`ActionKit <https://actionkit.com/>`_ is a platform for advocacy, fundraising, and
+get-out-the-vote. This parsons integration with the
```
Comment (eliotst) [inline docs/action_kit.rst:9]: One small thing: we capitalize "Parsons" when referencing the name of the library in documentation.

File: docs/action_kit.rst [hunk 2]
```diff
@@ -5,19 +5,53 @@ ActionKit
...
+    to access the ActionKit API. See the `ActionKit REST API Authentication <https://roboticdogs.actionkit.com/docs/manual/api/rest/overview.html#authentication>`_
+    documentation for more information on obtaining ActionKit API credentials.
 
 **********
 Quickstart
 **********
 
-**Retrieve a User**
+To instantiate the ActionKit class, you can either store your ActionKit API
+domain, username, and password as environmental variables (ACTION_KIT_DOMAIN,
```
Comment (eliotst) [inline docs/action_kit.rst:26]: Another minor thing.. could you wrap the environment variables' names in double backticks (e.g. \``ACTION_KIT_DOMAIN\``) so they are formatted as code?

**Comment by shaunagm** [comment]
Thanks so much @rgriff23! I updated issue 269 to use the actionkit connector as the new model.  :)

**Comment by rgriff23** [comment]
> For creating docs internally you need two packages:
> 
> ```
> Sphinx==1.8.3
> sphinx-rtd-theme==0.4.2
> ```
> 
> Then to build do the following:
> 
> ```
> > cd docs
> > make html
> ```
> 
> It will generate all of the html locally that you can then inspect. If there are recommendations as to how to better systematize this, I'm open to it. The html files shouldn't be included in the commit though, as they are built on deploy.

@jburchard @shaunagm I also had to install `m2r` for this to work

**Pull request merged**